certora/makefile

```diff
@@ -7,15 +7,15 @@ munged-simple: $(wildcard ../src/*.sol) applyHarnessSimple.patch
 	@patch -p0 -d munged-simple < applyHarnessSimple.patch
 
 record-simple:
-	diff -ruN ../src munged-simple | sed 's+\.\./src/++g' | sed 's+munged-simple/++g' > applyHarnessSimple.patch
+	diff -ruN ../src munged-simple | sed 's+\.\./src/++g' | sed 's+munged-simple/++g' | sed 's,\(\-\-\- [^[:space:]]*\).*,\1,' | sed 's,\(+++ [^[:space:]]*\).*,\1,' > applyHarnessSimple.patch
```

Comment:
Can't we merge this sed filter in a single one? Something along the lines of 
```
sed 's,\(\-\-\- [^[:space:]]*\).*\|\(+++ [^[:space:]]*\).*,\1\2,'
```

Reply:
I think we can, but I'd say it's harder to read and we don't care much about efficiency for this

Reply:
I think the following works too: `sed 's,\(\(\-\-\-\|+++\) [^[:space:]]*\).*,\1,'`. Do you like it better than the current one ?

Reply:
Yes but is optional do what you thinks is best :) Personally I think your latest proposition is the most readable. In any case, sed filters are not known for their readability 😅!